spec/preload.lua

```diff
@@ -1,12 +1,15 @@
 -- This script is given to Busted via the --helper argument. Modules loaded here
 -- won't be cleared and reloaded by Busted. This is needed for lgi because lgi
 -- is not safe to reload and yet Busted manages to do this.
-require("lgi")
+require "lgi"
```

Comment:
any reason for this?

Reply:
No particular reason. I think it's from my formatter and I was too lazy to not git-commit it.

Reply:
let's just try to keep it uniform